lib/src/util/vulnerabilitySeverityRatings.ts

```diff
@@ -67,10 +69,9 @@ export const getVulnerabilitySeverityRating = (cvssScore: VulnerabilityScore | n
  * (Internal use, not exported to library users)
  * Maps vulnerability CVSS severity ratings to their corresponding score ranges string representation.
  */
-export const vulnerabilitySeverityScoreRangeMap = Object.freeze({
-  critical: '9.0—10.0',
-  high: '7.0—8.9',
-  medium: '4.0—6.9',
-  low: '0.1—3.9',
-  none: '0.0'
-});
+export const vulnerabilitySeverityScoreRangeMap = Object.freeze(
+    vulnerabilitySeverityAndScoreRangeList.reduce((acc, range) => {
+      acc[range.severity] = `${range.min}—${range.max}`;
```

Comment:
This isn't quite equivalent.  The `none` case (or any case where the min and max are the same) might need to be handled specially.

Reply:
> This isn't quite equivalent. The `none` case (or any case where the min and max are the same) might need to be handled specially.

Fixed that here: [9642cc5](https://github.com/sonatype/sonatype-react-shared-components/pull/1202/commits/9642cc5fe1ab3e31c3f982313d993970c398730c)